infra_tooling_account/infra_tooling_account/infra_tooling_common_stack.py

```diff
@@ -112,7 +112,9 @@ def create_settings_bucket(self) -> s3.Bucket:
         settings_bucket = s3.Bucket(
             self,
             "salmonSettingsBucket",
-            bucket_name=AWSNaming.S3Bucket(self, "settings"),
+            bucket_name=AWSNaming.S3Bucket(
+                self, "settings", NestedStack.of(self).account
```

Comment:
I'm wondering if it might be easier just pass
"settings-" + account_id as meaning param value and not to change awsnaming param list?
What do you think?

Reply:
sure, good idea, let me adjust it
thank you!